### PR TITLE
[no ticket][risk=low] update tables to latest

### PR DESCRIPTION
**Pull request by jaycarlton**
Bring all configs up-to-date, which is still slightly behind, but a good stopping point. We should be able to add a new column now.

### CODE REVIEW

File: modules/workbench/providers.tf [hunk 1]
```diff
@@ -13,17 +13,17 @@ terraform {
...
-//  region  = "us-central1"
-//  zone    = "us-central1-c"
-//}
-
 provider "google" {
   version = "3.5.0"
-  project = var.project_id
-  region  = var.region
-  zone    = var.zone
+  project = "all-of-us-workbench-test"
```
Comment (calbach) [inline modules/workbench/providers.tf:18]: Was this for testing? Revert?

File: modules/workbench/variables.tf [hunk 2]
```diff
@@ -36,7 +36,62 @@ variable reporting_dataset_id {
...
-  description = "Access grants to the dataset"
-  type        = list(map(string)) # It's not possible to use optional attributes, so this is the tightest type
+  description = "Access list for the reporting dataset"
+  type        = list(map(string))
   default     = []
 }
+
+# TODO(jaycarlton) codegen this top-level variables as the union
+#   of all modules' variables files.
+# List of objects whose values correspond to the google_monitoring_notification_channel
```
Comment (calbach) [inline modules/workbench/variables.tf:46]: This stuff doesn't look relevant to the PR - remove? Minimally, should remove commented out code

**Review by calbach — APPROVED** [review]
I'll approve so you aren't blocked tomorrow, but please address my comments

File: modules/workbench/providers.tf (hunk 1 above)
Comment (jaycarlton) [reply modules/workbench/providers.tf:18]: You need it for `terraform import` to work in 0.13. Otherwise it tells you it doesn't know how to find the provider. Hopefully it can be removed soon.

File: modules/workbench/variables.tf (hunk 2 above)
Comment (jaycarlton) [reply modules/workbench/variables.tf:46]: yeah, thought I I needed it, but I guess not.

File: modules/workbench/providers.tf (hunk 1 above)
Comment (calbach) [reply modules/workbench/providers.tf:18]: I don't follow, how does this get overridden when we use this module for an environment that is not all-of-us-workbench-test ? If it's not getting overridden, and the module still works properly for other projects, what is this parameter actually doing then?

Comment (jaycarlton) [reply modules/workbench/providers.tf:18]: Good point. I think the right way to do it is to specify the provider only at the highest level.
